tests/basics/list_slice_3arg.py

```diff
@@ -26,3 +26,10 @@
 print(x[-1:-2:-1])
 print(x[-1:-11:-1])
 print(x[-10:-11:-1])
+
+print([][::-1])
+print([1][::-1])
+print([][:-20:-1])
+print([1][:-20:-1])
+print([][-20::-1])
+print([1][-20::-1])
```

Comment:
Thanks for that feedback.  I've updated the test with your recommendation (showing the error) and also moved the crashing tests behind an `if`.